Gemfile

```diff
@@ -103,10 +103,10 @@ group :development, :test do
   gem 'net-http-persistent'
   gem 'rinku'
   gem 'rspec'
-  gem 'selenium-webdriver', '3.141.0'
+  gem 'selenium-webdriver', '~> 4.2.0' # 4.x is required for Ruby 3 support; <4.3 required for Http:Persistent to not be deprecated yet
   gem 'spring', '~> 3.1.1'
   gem 'spring-commands-testunit'
-  gem 'webdrivers', '~> 3.0'
+  gem 'webdrivers', '~> 4.7' # 4.7 required for selenium 4.x
```

Comment:
Would you consider updating all the way to webdrivers ~> 5.2? This would help with arm64 support, otherwise I'll probably do a followup PR to update all the way to 5.2 😆

Reply:
Definitely! For both this and updating selenium past 4.3, once I get things working on this version I'm going to target more-recent versions and very possibly deploy all updates at once. I just want to move slowly so I know exactly where each of the various test failures I'm running into is coming from.